### DISCUSSION TITLE
Meeting 2024-09-03

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#790] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-09-03)

### Announcements
* None yet

**Comment by jamesmunns** [comment]
I have opened https://github.com/rust-embedded/wg/issues/792 - to discuss looking at `build-std` as an alternative to "target proliferation" for Arm v8m and RISC-V extension support. It would be good to discuss briefly, and get feedback from folks that have opinions on this, or had discussions at RustNL or elsewhere about this topic.

**Comment by jamesmunns** [comment]
Brief council announcement:

* Project Director elections are coming soon, members of the embedded wg can nominate directors (blog post coming soon: https://github.com/rust-lang/blog.rust-lang.org/pull/1388)
* Rust All Hands is going to be in NL next year, the "save the date" is here: https://blog.rust-lang.org/inside-rust/2024/09/02/all-hands.html, there is also a survey to indicate your interest if you are an embedded-wg team member

**Comment by jannic** [comment]
https://github.com/rust-embedded/cortex-m/pull/555 needs a review and looks quite obvious to me.

**Reply by adamgreig** [reply]
thanks for highlighting it, done!

**Comment by adamgreig** [comment]
I'll be absent for the meetings on the 17th, 24th Sept and 1st Oct, so someone else will need to run the meetings on those days.
